ui/src/main/webapp/components/FileInputComponent/FileInputComponent.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import FileInputComponent from './FileInputComponent';
+import BaseFormView from '../BaseFormView';
+
+const meta = {
+    component: FileInputComponent,
+    title: 'Components/FileInputComponent',
+    argTypes: { handleChange: { action: 'input file changed' } },
+} satisfies Meta<typeof BaseFormView>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+const common = {
+    field: 'fileInptuComponent',
+    controlOptions: {
+        fileSupportMessage: 'test support message',
+        supportedFileTypes: ['json', 'txt'],
+        maxFileSize: 500,
+    },
+    disabled: false,
+};
+
+export const Base: Story = {
+    args: common,
```

Comment:
we gets loose typing by declaring props that way. Please declare it inline and reuse it like `Base.args`